guide/src/wasm-bindgen-test/continuous-integration.md

```diff
@@ -10,37 +10,39 @@ request!](https://github.com/rustwasm/wasm-bindgen)
 
 ```yaml
 language: rust
-rust: nightly
+rust    : nightly
 
 addons:
   firefox: latest
-  chrome: stable
+  chrome : stable
 
 install:
-  - rustup target add wasm32-unknown-unknown
-  # Downloads a `wasm-bindgen` release binary from https://github.com/rustwasm/wasm-bindgen/releases.
-  # Alternatively, use `wasm-pack` to manage `wasm-bindgen` binaries for you
-  - curl -OL https://github.com/rustwasm/wasm-bindgen/releases/download/0.2.21/wasm-bindgen-0.2.21-x86_64-unknown-linux-musl.tar.gz
-  - tar xf wasm-bindgen-0.2.21-x86_64-unknown-linux-musl.tar.gz
-  - chmod +x wasm-bindgen-0.2.21-x86_64-unknown-linux-musl/wasm-bindgen
-  # Moves the binaries to a directory that is in your PATH
-  - mv wasm-bindgen-0.2.21-x86_64-unknown-linux-musl/wasm-bindgen* ~/.cargo/bin 
-  # Install node.js with nvm.
-  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
-  - source ~/.nvm/nvm.sh
-  - nvm install v10.5
-  # Install chromedriver.
-  - curl --retry 5 -LO https://chromedriver.storage.googleapis.com/2.41/chromedriver_linux64.zip
-  - unzip chromedriver_linux64.zip
-  # Install geckodriver.
-  - curl --retry 5 -LO https://github.com/mozilla/geckodriver/releases/download/v0.21.0/geckodriver-v0.21.0-linux64.tar.gz
-  - tar xf geckodriver-v0.21.0-linux64.tar.gz
+  - curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
 
 script:
-  # Test in Chrome.
-  - CHROMEDRIVER=$(pwd)/chromedriver cargo test --target wasm32-unknown-unknown
-  # Test in Firefox.
-  - GECKODRIVER=$(pwd)/geckodriver cargo test --target wasm32-unknown-unknown
+
+  # this will test the non wasm targets if your crate has those, otherwise remove this line.
+  #
+  - cargo test
+
+  - wasm-pack test --firefox --headless
+  - wasm-pack test --chrome  --headless
+```
+
+Note that the wasm-pack installer will ask for confirmation before overwriting the wasm-pack binary. So if you use caching you can make sure it's not caching wasm-pack by adding this to your `travis.yml`:
```

Comment:
I think this note can be removed by passing `-y` to the installer script?

Reply:
I'll try that, when looking over the script I didn't see any argument parsing, so I didn't try that. I also didn't see any usage information.

I tried several things without success: 
```
curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf > wasm-pack-installer.sh
chmod +x wasm-pack-installer.sh
./wasm-pack-installer.sh -y
./wasm-pack-installer.sh -h
yes | ./wasm-pack-installer.sh
```

Reply:
I think `curl https://rustwasm.github.io/wasm-pack/installer/init.sh | sh -s -- -y` should do the trick

Reply:
Er sorry looks like the script doesn't actually support `-y` (now I'm actually reading it)

Hm for now can you open an issue in the wasm-pack repo? I'd prefer to avoid weird gotchas in the documentation like this

Reply:
I have opened rustwasm/wasm-pack#668.

Personally I feel it's more honest towards users to remove gotchas from the documentation once the issue is resolved, but it's your call.

Reply:
Thanks for that! I agree it's good to be truthful but I don't think that this is the place to have exhaustive documentation about everything. You already have to opt-in to caching and caching already has its own set of issues independent of this, so I would prefer that we delete this section with caveats about caching.